assets/prototype/domain/eventEditor/containers/useConfigProvider.ts

```diff
@@ -1,30 +1,25 @@
 import { useContext } from 'react';
-import { pick } from 'ramda';
 import { ConfigContext } from '../../../infrastructure/services/contextProviders/ConfigProvider';
+import getDateTimeFormat from '../../../application/valueObjects/dateTime/getDateTimeFormat';
 
 type useConfigProviderProps = {
-	dateFormat: boolean;
-	email: boolean;
-	timezone: boolean;
+	dateTimeFormat: boolean;
 };
 
 /**
  *
  * useConfigProvider is used to cherry-pick the exact configs we might need from ConfigProvider.
  */
+const useConfigProvider = ({ dateTimeFormat }: useConfigProviderProps) => {
+	const { dateFormat, timeFormat } = useContext(ConfigContext);
+	let config: any = {};
 
-const useConfigProvider = (props: useConfigProviderProps) => {
-	const config = useContext(ConfigContext);
+	if (dateTimeFormat) {
+		const dateTimeFormat = getDateTimeFormat({ dateFormat, timeFormat });
+		config = { ...config, ...dateTimeFormat };
+	}
```

Comment:
@tn3rb this is the result, maybe I'll refactor the approach a bit as new information will come in

Reply:
if nothing will be passed before `if (dateTimeFormat) {`
the `config` will be:
```
config = {
        dateFormat,
		dateTimeFormat,
		timeFormat,
}
```
again, I am not married to this approach, this is just the simplest way to pass data around